apis/fluentbit/v1alpha2/plugins/output/azure_blob_types.go

```diff
@@ -19,22 +17,26 @@ type AzureBlob struct {
 	// Name of the container that will contain the blobs
 	ContainerName string `json:"containerName"`
 	// Specify the desired blob type. Must be `appendblob` or `blockblob`
+	// +kubebuilder:validation:Enum:=appendblob;blockblob
 	BlobType string `json:"blobType,omitempty"`
 	// Creates container if ContainerName is not set.
-	AutoCreateContainer *bool `json:"autoCreateContainer,omitempty"`
+	// +kubebuilder:validation:Enum:=on;off
+	AutoCreateContainer string `json:"autoCreateContainer,omitempty"`
 	// Optional path to store the blobs.
 	Path string `json:"path,omitempty"`
 	// Optional toggle to use an Azure emulator
-	EmulatorMode *bool `json:"emulatorMode,omitempty"`
+	// +kubebuilder:validation:Enum:=on;off
+	EmulatorMode string `json:"emulatorMode,omitempty"`
 	// HTTP Service of the endpoint (if using EmulatorMode)
 	Endpoint string `json:"endpoint,omitempty"`
 	// Enable/Disable TLS Encryption. Azure services require TLS to be enabled.
-	TLS *bool `json:"tls,omitempty"`
+	// +kubebuilder:validation:Enum:=on;off
+	TLS string `json:"tls,omitempty"`
```

Comment:
`tls` can be written in such a way that this ：https://github.com/fluent/fluent-operator/blob/master/apis/fluentbit/v1alpha2/plugins/output/elasticsearch_types.go#L97

Reply:
fixed